pkg/p2p/wire/message/candidate.go

```diff
@@ -6,24 +6,19 @@ import (
 	"strings"
 
 	"github.com/dusk-network/dusk-blockchain/pkg/core/consensus/header"
-	"github.com/dusk-network/dusk-blockchain/pkg/core/consensus/key"
-	"github.com/dusk-network/dusk-blockchain/pkg/core/consensus/user"
 	"github.com/dusk-network/dusk-blockchain/pkg/core/data/block"
 	"github.com/dusk-network/dusk-blockchain/pkg/p2p/wire/message/payload"
-	"github.com/dusk-network/dusk-blockchain/pkg/util"
 )
 
 // Candidate is the composition of block and certificates
 type Candidate struct {
```

Comment:
I guess the Candidate is just *block.Block. No need to add yet another type for the same thing unless there is some compelling reasons I am not seeing